src/map/battle.cpp

```diff
@@ -8160,6 +8162,9 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 
 	if (!flag.infdef) { //No need to do the math for plants
 		uint32 skillratio = 100; //Skill dmg modifiers.
+		if (sd != nullptr)
+			skillratio += sd->bonus.skill_ratio;
```

Comment:
Is it really okay to apply this here for renewal?
See has_skillratio below.

Reply:
Yes, that should be fine. Skills that don't make use of skillratio are simply not affected then. Or should we force them to be affected by it then?

Reply:
Ok perfect. Just didnt know if you saw that.